wiki/Help_centre/Store/en.md

```diff
@@ -2,124 +2,48 @@
-
-- Unscrew the grip from the body of the pen, exposing the battery on the inside.
-- Remove the AAA battery from the pen.
-- Replace the battery with a brand new AAA battery. **Verify that the new battery functions in another device first.**
-- Ensure that the battery's positive and negative ends are appropriate in the pen. There are markers on the device that display marks telling you which end goes where.
-- Screw the grip back onto the pen.
-- Depress the button on the back end (eraser end) of the pen until it clicks.
-
-If your tablet unit is functioning properly, placing the pen near the tablet will make the cursor on your screen move.
-
-Please contact [store@ppy.sh](mailto:store@ppy.sh) for further inquiries.
-
-## osu!go {id=osu!go}
-
-### My computer doesn't recognise the osu!go when I plug it in! {id=osu!go-not-recognised}
-
-**This is a known issue with one of the very first shipments of the osu!go USB sticks, and comes down to the device not being formatted in a manner that works with all PCs.**
-
-Despite how complex this may sound, it's actually very easy to fix.
-
-First off, [download the package containing the osu!go default files](https://assets.ppy.sh/store/utilities/osu!go.zip) and save it somewhere on your computer.
-
-#### Formatting the osu!go under Windows {id=osu!go-format}
-
-**Follow these steps to prepare your osu!go device under Windows:**
-
-1. Right click the `Start` menu button and select `Disk Management`.
-2. **While in the Disk Management view, make sure to follow these instructions *VERY* closely and ONLY apply them to the osu!go device.** You can potentially lose data if you fiddle around in this dialog.
-3. Check to see what state your computer recognises the device as being in.
-
-### The osu!go device is present, but in a "RAW" state {id=osu!go-raw}
-
-**This will appear like this in the Disk Management dialog:**
-
-![](img/raw-status-osu-go.png)
-
-To fix this, ensure you have the osu!go device selected, right-click on the RAW partition, and select the `Format` option. Choose either `NTFS` or `exFAT` under the `File System` dropdown.
-
-`exFAT` will work with more devices, but is slightly slower.
-
-### The osu!go device doesn't show up in Disk Management at all {id=osu!go-partition}
-
-**You will need to partition the device from scratch.**
-
-Please follow [this guide](https://tails.boum.org/doc/reset/windows/index.en.html) for details on how to do that.
-
-### I've reformatted the osu!go device and it now shows up in Disk Management {id=osu!go-clean}
-
-**You're almost done!**
-
-Unzip the archive we asked you to download earlier (osu!go.zip) into the osu!go drive and you're done!
-
-### None of this worked! {id=osu!go-unknown-cause}
-
-If you encounter any issues with this process, please send an email to [store@ppy.sh](mailto:store@ppy.sh) and we'd be happy to help you out.
+That sucks. Contact [merch@ppy.sh](mailto:merch@ppy.sh) and we'll try to figure it out.
```

Comment:
"that sucks" feels kind of out of place but idk, I can tell you wanted this page to feel more casual in tone than the average customer support page so maybe it's fine